modules/motions/hooks/useTransitionLimits.ts

```diff
@@ -7,56 +7,75 @@ import {
   ContractEVMScriptExecutor,
 } from 'modules/blockChain/contracts'
 
-import { BigNumber, utils } from 'ethers'
+import { utils, constants } from 'ethers'
 import { Big } from 'modules/shared/utils/bigNumber'
 
+// Data structure reference
+// https://github.com/lidofinance/scripts/blob/2a30b9654abc90b20debf837f99cd02f248d6644/scripts/setup_easytrack_limits.py#L67-L100
+const TOKEN_INDEXES = [
```

Comment:
Could we get rid of comments here?
for example:
```
const LDO_INDEX = 1
const ETH_INDEX  = 4 <- and use it second time in row 65 instead of "4"
....
TOKEN_INDEXES = [LDO_INDEX, ETH_INDEX, ...]
```